src/sdk.js

```diff
@@ -243,6 +247,10 @@ class Smartcar {
 
     link += `&mode=${this.mode}`;
 
+    if (options.singleSelect) {
```

Comment:
should we add the check here ``` if (options.singleSelect === true) {``` And then in the next line just do `${options.singleSelect}`

Reply:
I think both approaches are valid. I decided to write it this way because my logic was that if the developer specifies any value for `singleSelect`, we should include it in the URL.